src/components/Catalyst/Step1.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+/**
+ * Step 1 for the Catalyst registration - landing
```

Comment:
Just a suggestion for the future: I think naming files/steps with numbers is not always a good a idea because steps can be introduced or removed at random places in time, which messes all the naming up. Better to use a name like `Landing.js` or something more descriptive.